tasks/agent.py

```diff
@@ -44,14 +44,18 @@
 @task
 def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None,
           puppy=False, use_embedded_libs=False, development=True, precompile_only=False,
-          skip_assets=False):
+          skip_assets=False, android=False):
```

Comment:
Why do we need to update `agent.build` command if we have the `android.build` one ?

Reply:
good catch... I started with build, but decided to make its own command.  Now fixed.